packages/typespec-azure-resource-manager/test/rules/missing-x-ms-identifiers.test.ts

```diff
@@ -130,4 +130,70 @@ describe("typespec-azure-core: no-enum rule", () => {
       )
       .toBeValid();
   });
+
+  it("allow array of x-ms-identifiers", async () => {
+    await tester
+      .expect(
+        `
+        model Pet {
+          @OpenAPI.extension("x-ms-identifiers", ["food/brand/name"])
```

Comment:
I think we should also test an identifier that starts with `/` (i.e. /food/brand/name)